src/shaders/lighting.wgsl

```diff
@@ -38,7 +39,7 @@ fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
     let light_count = MAX_LIGHTS;
 #endif
 
-    for (var i = 0u; i < light_count; i++) {
+    for (var i = 0u; i < buffer_size.size; i++) {
```

Comment:
Since we are adding the buffer size explicitly, we don't need to keep track of the `lightCount` variable, that can be deleted entirely.